charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -202,6 +202,46 @@ scrape_configs:
     action: replace
     replacement: ${1}
 
+- job_name: 'kube-system/kubelet'
+  kubernetes_sd_configs:
+  - role: node
+  relabel_configs:
+  - action: labelmap
+    regex: __meta_kubernetes_node_label_(.+)
+  - target_label: component
+    replacement: kubelet
+  - action: replace
+    source_labels: [__meta_kubernetes_node_name]
+    target_label: instance
+  - source_labels: [__address__]
+    action: replace
+    target_label: __address__
+    regex: ([^:;]+):(\d+)
+    replacement: ${1}:10255
+  - source_labels: [__scheme__]
+    action: replace
+    target_label: __scheme__
+    regex: https
+    replacement: http
+
+- job_name: 'kubernetes-cadvisors'
+  scheme: https
+  tls_config:
+    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    insecure_skip_verify: true
```

Comment:
why? Can we get this fixed?

Reply:
Seems like we don't have `kubernetes.default.svc` in the added to the apiservers SANs. `kubernetes.default` should work.